apps/web/src/constants/tokens.ts

```diff
@@ -522,6 +554,9 @@ export const TOKEN_SHORTHANDS: { [shorthand: string]: { [chainId in ChainId]?: s
     [ChainId.SEPOLIA]: USDC_SEPOLIA.address,
     [ChainId.AVALANCHE]: USDC_AVALANCHE.address,
   },
+  USDT: {
```

Comment:
should we add USDC a few lines above?

Reply:
I'll have this clean and add USDC in an upcoming PR